invoke-ai/InvokeAI
# fix(ui): model manager goes blank

## Summary

The model edit UI's composition allows for the model edit form to be instantiated before the model's config has been received. This results in the form having no values - all the fields are blank instead of populated by the model config.

Part of the fix is to pass the model config around directly instead of relying on _all_ components to fetch the model directly.

I also fixed a crapload of performance issues related to improper use of redux selectors.

## Related Issues / Discussions

Closes #6347

## QA Instructions

To reproduce the issue:
- Go to the model manager
- Select a model
- Click Edit
- Click the Generation tab
- Wait at least 60 seconds (this clears the cached model config, which will trigger the race condition)
- Return to the model manager tab

Also, model manager UI should feel substantially snappier.

## Merge Plan

n/a

## Checklist
